Responsive coverage and Desktop layout fixes

DimaMklk opened the pull request.

Commit:
Responsive coverage and Desktop layout fixes

Commit:
Responsive Fixes

landing's footer align centring and removing margin-top from userPorfilePage

barbarosh commented on client/components/TalentIcon/TalentIcon.js:
```diff
@@ -5,7 +5,9 @@ import ReactTooltip from 'react-tooltip';
 function TalentIcon(props) {
-  let classes = cn(styles.icon, {
+  let classes = cn({
+    [styles.icon]: !props.mobileEvaluation,
```
Зачем тут убираешь главный стиль ?

DimaMklk replied:
@barbarosh в мобильном леяуте evaluates иконки без стилей который описаны в класе icon. лишь изменения размеров

barbarosh commented on client/components/TalentIcon/TalentIcon.js:
```diff
@@ -5,7 +5,9 @@ import ReactTooltip from 'react-tooltip';
-  let classes = cn(styles.icon, {
+  let classes = cn({
+    [styles.icon]: !props.mobileEvaluation,
+    [styles.mobileEvaluation]: props.mobileEvaluation,
```
Вместо флага mobileEvaluation используй медиа кверис в стилях

DimaMklk replied:
@barbarosh не могу, так как тогда изменяться стили и в setup profile например. а там мне нужны стили класа icon

barbarosh replied:
Понял, почему так, но нужно переделать, в таком случае класс икон переименуй в circled, а проп mobileEvaluation в circled и переделай код так чтобы это была не функция а класс и проп cirсled по-умолчанию была true, а класс mobileEvaluation  переименуй  в icon и  в итоге classes будет выглядеть так 
'''
let classes = cn(styles.icon, {
    [styles.circled]: props.circled,
    [styles['icon--selected']]: props.selected,
    [styles['icon--enabled']]: !!props.onClick
    });
'''

barbarosh commented on client/modules/Evaluate/components/EvaluateTalentsForm/EvaluateTalentsForm.js:
```diff
@@ -56,7 +62,12 @@ export class EvaluateTalentsForm extends Component {
-          <div className={styles.anonymous}>This review will be fully anonymous</div>
+          <div className={styles.anonymous}>
+            <MediaQuery query='(max-width: 767px)'>
+              <i className="fa fa-lock" aria-hidden="true"></i>
```
Забыл про это, спасибо

Commit:
TalentIcon, DiscItem and TalentsForm Fixes

TalentIcon circled state.
Response DiscItem's plus/minus states.
Pesponse TalentsForm's layout.

Commit:
DiscItem's plus/minus default styles and desktop style correction

Commit:
Media query styles and RadarCharts image positioning correction
